tests/sentry/api/endpoints/test_organization_sentry_app_installations.py

```diff
@@ -190,3 +193,21 @@ def test_cannot_install_other_org_internal_app(self):
         org2 = self.create_organization()
         internal_app = self.create_internal_integration(name="Internal App", organization=org2)
         self.get_error_response(self.org.slug, slug=internal_app.slug, status_code=404)
+
+    @with_feature({"organizations:integrations-alert-rule": False})
+    def test_disallow_app_with_all_features_disabled(self):
```

Comment:
Do we also have tests for successfully installing a sentryapp owned by another org? I'm only seeing tests for installing apps owned by the same organization as the organization doing the install.

Reply:
Added the test `test_install_published_app_by_other_org` above.